package com.example.rqchallenge.exception;

public class ErrorMessage {

    public static final String EXCEPTION_OCCURRED_WHILE_INTERACTING_WITH_RESOURCE = "Error occurred interacting with resource {}";
    public static final String EXCEPTION_WHILE_FETCHING_EMPLOYEES = "Exception occurred interacting with external service to fetch employees";
    public static final String EXCEPTION_OCCURRED_WHILE_DELETING_EMPLOYEE = "Exception occurred deleting employee with id: %s";
    public static final String EXCEPTION_WHILE_CREATING_EMPLOYEE = "Exception occurred while creating employee";
    public static final String EXCEPTION_OCCURRED_WHILE_FINDING_EMPLOYEE = "Exception occurred while fetching employee with id: %s";
}
